src/index.html

```diff
@@ -22,6 +22,394 @@
   </head>
 
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a
+        href="/"
+        class="logo"
+      >
+        <img
+          src="/src/images/logo.png"
+          alt="MOYO logo"
+        />
+      </a>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="li">
+            <a
+              href="#"
+              class="li__link is-active"
```

Comment:
Fix classes everywhere
```suggestion
          <li class="nav__item">
            <a
              href="#"
              class="nav__link is-active"
```